ArduCopter/Copter.h

```diff
@@ -368,6 +368,7 @@ class Copter : public AP_Vehicle {
             uint8_t unused3                 : 1; // 25      // was compass_init_location; true when the compass's initial location has been set
             uint8_t unused2                 : 1; // 26      // aux switch rc_override is allowed
             uint8_t armed_with_switch       : 1; // 27      // we armed using a arming switch
+            uint8_t air_mode                : 2; // 28      // air mode is 0 = not-configured ; 1 = disabled; 2 = enabled
```

Comment:
as discussed on the dev call, let's move this to a separate value and change the type to AirMode